src/pretalx/schedule/models/schedule.py

```diff
@@ -143,9 +143,60 @@ def changes(self):
             result['action'] = 'create'
             return result
 
+        # ******************************************
+        # build list of all new talks without link to schedule
+        # from pretalx.submission.models import Submission
```

Comment:
I'd still like to ask you to remove the debug info – but this is definitely the most nicely formatted debug info I've come across recently! :wink:

Reply:
thanks ;-)
i called this commit WIP as i just wanted to get this stored...
but i know i had to clean this up a lot ;-)
iam now onto this..

Reply:
done :-)